src/presentation/test/integration/registration/what-is-your-jurisdiction.test.ts

```diff
@@ -0,0 +1,148 @@
+import request from "supertest";
+import {
+  PartnershipType
+} from "@companieshouse/api-sdk-node/dist/services/limited-partnerships";
+
+import enTranslationText from "../../../../../locales/en/translations.json";
+import cyTranslationText from "../../../../../locales/cy/translations.json";
+import app from "../app";
+import { getUrl, setLocalesEnabled } from "../../utils";
+import { appDevDependencies } from "../../../../config/dev-dependencies";
+import { APPLICATION_CACHE_KEY_PREFIX_REGISTRATION } from "../../../../config/constants";
+import RegistrationPageType from "../../../controller/registration/PageType";
+import {
+  WHAT_IS_YOUR_JURISDICTION_URL
+} from "../../../controller/registration/url";
+
+describe("Which jurisdiction page", () => {
+
+  const URL = getUrl(WHAT_IS_YOUR_JURISDICTION_URL);
+
+  beforeEach(() => {
+    setLocalesEnabled(false);
+  });
+
+  it("should load the name page with English text for EnglandWales LP jurisdiction", async () => {
```

Comment:
your tests say they are testing the name page, should be jurisdiction page

Reply:
Renamed